### ISSUE TITLE
sass-loader v4 breaks with Unexpected character ' '

### ISSUE THREAD

**Issue by BerndWessels** [jtangelder/sass-loader#264]
Hi

I am using icomoon fonts in my project.
With sass-loader v3 everything works fine.
With sass-loader v4 it breaks with `Unexpected character ' '`

You should be easily able to recreate it by just adding an icomoon font to your test project and see how it breaks with sass-loader v4.

Let me know if you can't reproduce it and need more details.

**Comment by jhnns** [comment]
It would be very helpful if you could setup a small example project which I can clone and run. Your description is still a bit vague.

**Comment by BerndWessels** [comment]
@jhnns Hi, thanks for the quick response.

In the little example project I built to make this issue reproducable for you the error didn't show up.

So I updated all other dependencies in my original project and did a fresh `npm install` which fixed the issue.

So the problem must have been an incompatibility between `sass-loader` v4 and one of my other `outdated` dependencies.

In case somebody has a similar error, this might help to fix it.

Closing this now, all good, thanks.
